src/modules/selection/RectangularSelection.jsx

```
@@ -0,0 +1,248 @@
+   */
+  const handleSelect = useCallback(
+    e => {
+      const isMultiSelect = e.inputEvent?.ctrlKey || e.inputEvent?.metaKey
+      const newSelection = isMultiSelect ? { ...selectedItems } : {}
+      let count = isMultiSelect ? Object.keys(selectedItems).length : 0
+
+      for (const el of e.selected) {
+        const file = getFileFromElement(el)
+        if (file && !newSelection[file._id]) {
+          newSelection[file._id] = file
+          count++
```

ESLint

[error] 'count' is assigned a value but never used.

(no-unused-vars)